OrbitCore/SamplingProfiler.cpp

```diff
@@ -348,22 +350,25 @@ std::multimap<int, CallstackID> ThreadSampleData::SortCallstacks(
 }
 
 //-----------------------------------------------------------------------------
-void SamplingProfiler::ProcessAddresses() {
+void SamplingProfiler::ResolveCallstacks() {
   ScopeLock lock(m_Mutex);
   for (const auto& it : m_UniqueCallstacks) {
     CallstackID rawCallstackId = it.first;
     const std::shared_ptr<CallStack> callstack = it.second;
+    // A "resolved callstack" is a callstack where every address is replaced by
+    // the start address of the function (if known).
     CallStack resolvedCallstack = *callstack;
 
     for (uint32_t i = 0; i < callstack->m_Depth; ++i) {
       uint64_t addr = callstack->m_Data[i];
 
-      if (m_ExactAddresses.find(addr) == m_ExactAddresses.end()) {
+      if (m_ExactAddressToFunctionAddress.find(addr) ==
+          m_ExactAddressToFunctionAddress.end()) {
         AddAddress(addr);
       }
 
-      auto addrIt = m_ExactAddresses.find(addr);
-      if (addrIt != m_ExactAddresses.end()) {
+      auto addrIt = m_ExactAddressToFunctionAddress.find(addr);
```

Comment:
since you are renaming it, maybe follow google style?

Reply:
The point is that either I follow the current style, or I rename every field (it's a lot of them), and I don't really want to do it in these kinds of focused changes.